.github/workflows/rgdc.yml

```
@@ -0,0 +1,35 @@
+name: rgdc python client
+on:
+  push:
+    branches: "*"
+    tags: "v*"
+  pull_request:
+    branches: "**"
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Install rgdc
+        working-directory: ./rgdc
+        run: pip install -e .
+      - name: Test (not implemented)
+        working-directory: ./rgdc
+        run: python -c "from rgdc import Rgdc; print('Import went okay!')"
+      - name: Build and Publish to PyPI
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+        working-directory: ./rgdc
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

We should probably use api tokens for this, in which case this would become something like
```suggestion
          TWINE_USERNAME: __token__
          TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```

---

Ah okay, I will remove those secrets and add a new token

---

Added my token as a secret